.env.j2

```diff
@@ -0,0 +1,17 @@
+FLASK_APP=todo_app/app
+
+FLASK_DEBUG=true
+
+SECRET_KEY={{secret_key}}
+
+TRELLO_API_KEY={{trello_api_key}}
+
+TRELLO_API_SECRET={{trello_api_secret}}
+
+TRELLO_API_TOKEN={{trello_api_token}}
+
+TRELLO_BOARD_ID='65f065616e29fcb606bf5717'
+
+TO_DO_LIST_ID='65f065616e29fcb606bf571e'
+
+DONE_LIST_ID='65f065616e29fcb606bf5720'
```

Comment:
Not that it matters here, but I'd default to entering your environment variables without quotes personally - whilst the `python-dotenv` library that handles this is forgiving either way, Docker is less helpful and tends to interpret the quotes as literally part of the variable, which can be a fiddly error to debug!